### DISCUSSION TITLE
Evidence of work, but no actual results

### DISCUSSION THREAD

**Discussion by nickwarrenwordsmith** [reworkd/AgentGPT#539] [Q&A]
I give it a goal and get an impressive amount of tasks added and lots of thinking and completing — but no actual results. Am I missing something? Do I need to ask for the actual results somehow?

**Comment by aaaclones** [comment]
No you are not alone here and the frustrating part about it all is it doesn't pick up where it left off it just loops continuously until it times out , I am unsure about this as well . Perhaps someone can chime in with some explanation , some pre prompts or prompts perhaps.
